src/backend/mpi/swdsm.cpp

```diff
@@ -326,8 +321,6 @@ void handler(int sig, siginfo_t *si, void *unused){
 
 	unsigned long * localAlignedAddr = (unsigned long *)((char*)startAddr + lineAddr);
 	unsigned long startIndex = getCacheIndex(lineAddr);
-	unsigned long cacheIndex = getCacheIndex(alignedDistrAddr);
-	cacheIndex = startIndex;
 	alignedDistrAddr = lineAddr;
```

Comment:
Not relevant to this PR, but this alias needs to disappear.

Reply:
This issue is addressed in #34